.github/workflows/docker-build-release.yml

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
@greglearns Instead of creating a separate `release` action, can you please consider merging this with https://github.com/crystaldba/crystaldba/blob/main/.github/workflows/release.yml?